adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -41,7 +41,7 @@
         }
```

Comment:
@javfg it seems like your IDE added a lot of empty lines and I can be wrong, be the only line that matters here is https://github.com/keycloak/keycloak/pull/6243/files#diff-f41a82d32ba240c5165fc1ee64741068R153. Could you please remove these empty lines?

Reply:
@abstractj I think his IDE was just removing extraneous spaces.